lib/tasks/webpacker/check_node.rake

```diff
@@ -29,6 +35,10 @@ namespace :webpacker do
       $stderr.puts "Node.js not installed. Please download and install Node.js https://nodejs.org/en/download/"
       $stderr.puts "Exiting!"
       exit!
+    rescue Errno::ENOSYS
+      $stderr.puts "'#{which_command}' is not installed. Please download and install '#{which_command}' by running the following command 'yum install #{which_command}'"
```

Comment:
Why should webpacker be responsible for ensuring items like `which`, `node`, and `yarn` are installed?

Could be better just to have the normal error when some required binary is not available?

@dhh @guillaumebriday  any thoughts?

Reply:
I agree. You could basically raise the standard error when a command does not exist.

There are here anyway https://github.com/rails/webpacker#prerequisites

Reply:
what is the standard error?

**the Prerequisites list doesn't mention the "which" binary**

not sure I'm following the conversation.

if you don't have "`which`" installed you get the "`Node.js not installed error`" message, that is not good.
This is why I am proposing a different error message in this PR for this scenario. a more accurate message will help to reduce the time to fix this problem. in my case I spent a couple hours trying to understand why nodejs was not installed and turns out that the issue was not related to nodejs at all I didn't have "which" installed